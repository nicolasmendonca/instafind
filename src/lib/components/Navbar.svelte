<nav class="bg-white flex justify-between p-4">
	<img src="/icons/hamburger.svg" alt="Menu" height="48" width="48" />

	<img src="/images/profile-placeholder.jpg" alt="Profile" height="48" width="48" />
</nav>
