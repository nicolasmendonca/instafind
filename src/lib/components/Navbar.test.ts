import Navbar from './Navbar.svelte'
import {render} from '@testing-library/svelte'

test('matches the snapshot', () => {
  const {container} = render(Navbar)
  expect(container).toMatchSnapshot()
})
