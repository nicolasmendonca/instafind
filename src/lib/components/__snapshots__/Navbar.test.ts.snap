// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<body>
  <div>
    <nav
      class="bg-white flex justify-between p-4"
    >
      <img
        alt="Menu"
        height="48"
        src="/icons/hamburger.svg"
        width="48"
      />
       
      <img
        alt="Profile"
        height="48"
        src="/images/profile-placeholder.jpg"
        width="48"
      />
    </nav>
  </div>
</body>
`;
